src/dispatch/plugins/dispatch_slack/commands.py

```diff
@@ -506,7 +542,26 @@ def list_incidents(
         projects.append(incident.project)
     else:
         # command was run in a non-incident conversation
-        projects = project_service.get_all(db_session=db_session)
+        if len(args) == 2:
+            project = project_service.get_by_name(db_session=db_session, name=args[1])
```

Comment:
`db_session` is scoped to the desired org here, correct?

Reply:
Yes, the organization db_session scoping is all taken care of before we ever enter any of the command functions.